content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

```diff
@@ -31,17 +39,163 @@ The Go Tracer requires Go `1.18+` and Datadog Agent `>= 5.21.1`. For a full list
 
 Before you begin, make sure you've already [installed and configured the Agent][5].
 
+There are two ways to instrument your Go application:
+
+1. **Manual instrumentation**:
+   - Gives you complete control over which parts of your application are traced.
+   - Requires modifying the application's source code.
+2. **Automatic compile-time instrumentation**:
+   - Ensures maximum coverage of your tracing instrumentation.
+   - Does not require source code modifications, making ideal for integrating at the CI/CD level.
+
+Refer to the instructions in the section corresponding to your preference below:
+
+{{% collapse-content title="Manual instrumentation" level="p" %}}
 ### Activate Go integrations to create spans
 
 Datadog has a series of pluggable packages which provide out-of-the-box support for instrumenting a series of libraries and frameworks. A list of these packages can be found in the [Compatibility Requirements][1] page. Import these packages into your application and follow the configuration instructions listed alongside each [Integration][1].
 
-## Configuration
+### Configuration
 
 If needed, configure the tracing library to send application performance telemetry data as you require, including setting up Unified Service Tagging. Read [Library Configuration][3] for details.
 
 For configuration instructions and details about using the API, see the Datadog [API documentation][4].
+{{% /collapse-content %}}
+
+{{% collapse-content title="Automatic compile-time instrumentation (alpha)" level="p" %}}
+
+<div class="card callout-card mb-4">
+  <div class="card-body d-flex flex-column">
+    <h5 class="card-title text-black mt-0 mb-1">Orchestrion is currently in alpha</h5>
+    <p class="card-text">
+      Orchestrion is under active development, and Datadog is eager to hear from you. Request access if you are interested and would like to share your experience.
+    </p>
+    <a href="TBD"
```

Comment:
TODO: Get link before merging.

Reply:
Yah that's literally the single reason why I've left this as "draft"... @vidurkhannadatadog should provide that form.